project1/styles.css

```
@@ -0,0 +1,174 @@
+/* FONTS
+	- default body text  mrs-eaves, "Palatino Linotype", "Book Antiqua", Palatino, serif;
+    - date & cite        mr-eaves-modern, "Futura", Helvetica, Arial, sans-serif;
+*/
+
+@import url("https://use.typekit.net/ckw4tit.css");
+
+/* HIGH-LEVEL SETTINGS */
+
+body {
+    font-family: mrs-eaves, "Palatino Linotype", "Book Antiqua", Palatino, serif;
+    font-weight: 400;
+    margin: 0 auto;
+    max-width: 1000px;
+    padding: 0 100px;
+}
+
+h1 {
+    display: flex;
+    font-size: 78px;
+    justify-content: center;
+    margin: 0;
+    padding: 50px 0 0 0;
+}
+
+p {
+    font-size: 22px;
+    /* size chosen to read well at max-width */
+}
+
+/* LOW-LEVEL SETTINGS */
+
+b {
+    font-feature-settings: "smcp";
+    /* makes the first line of each section display as smallcaps */
+}
+
+blockquote {
+    font-size: 75px;
+    font-style: italic;
+    font-variant-ligatures: discretionary-ligatures;
+    margin: 0 0 70px 0;
+    text-align: center;
+}
+
+cite {
+    display: flex;
+    font-family: mr-eaves-modern, "Futura", Helvetica, Arial, sans-serif;
+    font-size: 42px;
+    font-style: normal;
+    font-weight: 700;
+    justify-content: center;
+    margin: 0;
+    padding: 0;
+    text-align: center;
+    /* font-style must be set to normal here, or else browser defaults to italic */
+}
+
+div {
+    border: 4px solid;
+    margin: 25px;
+    padding: 50px;
+}
+
+
+/* CLASSES */
+
+.date {
+    display: flex;
+    font-family: mr-eaves-modern, "Futura", Helvetica, Arial, sans-serif;
+    font-size: 24px;
+    font-weight: 700;
+    justify-content: flex-end;
+    margin: 0;
+    padding: 40px 0 0 0;
```

Similar to the note about your `h1` styles above, I would avoid the `padding` shorthand because you really don't want or need to set all those zeros:
```suggestion
    padding-top: 40px;
```
Writing your CSS this way makes it clear what you're adding/changing without needlessly resetting the other padding values. (In writing my own CSS, I would choose to do the same for the `margin` rule on line 74, too. It's not much an issue on a smaller codebase like this, but on larger projects it is super helpful to just set the rules you intend to, so I'd write:
```css
margin-top: 0;
margin-bottom: 0;
```
over the margin shorthand.